### DISCUSSION TITLE
Manual update today (or changing the cron schema)

### DISCUSSION THREAD

**Discussion by mortgad** [dcaribou/transfermarkt-datasets#241] [Q&A] [answered]
Hi David, hope you are well

Once again i ask you to manually update the dataset :D Because the matches played this weekend are currently not a part of the data set and we need data from the weekend for our forecasting. 

Is that possible?

I can see this schema right now: 0 4 * * TUE,FRI
What is the explanation then for the weekend games not being part of the dataset? Is it because the web pages on TM are not yet updated?

**Comment by dcaribou** [comment, marked as answer]
Hi @mortgad,
The data should be updated in Kaggle by now.

The API token used by the pipelined had expired over the weekend.

**Reply by mortgad** [reply]
Thank you!
